### DISCUSSION TITLE
Major Drawback from GitHub I found While Verification

### DISCUSSION THREAD

**Discussion by MSubhajitIND** [community/community#128343] [GitHub Education] [answered]
### Hi everyone,

Problems I faced :

I can't upload my id card as it says "Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable."
Also, I found many drawback when ever I tried to submit. I told all this here and also told how i solved.

I submitted my ID card in official transcripts category and approved (at least in portal instantly approved but told to wait until final email come).

How i did :

1. I used [Camo Studio](https://reincubate.com/camo/) app (not sponsored) to capture image from phone to pc.
2. I simply split viewed my screen . One side was my github education application browser and the id opened image on other side.
3. Then I selected Camo camera from github application and pointed my phone to my id card side.
4. Then I captured the image and applied in official transcript category.

But there is major issues in this system which i had found :

1. Upload document option mission ... we have to take photo and it's hard to get a proper image of id.
2. Sometimes we have to upload both side of our id but we can upload only 1 image
3. In case of mobile phone user when you tried to flip camera it would not work. Front camera is very bad to upload document images also hard to take image of id properly. ( this issue is not solved more than 1 year ... i saw past tickets in portal )
4. Although there is no keyword in my ID which seems to be an acceptance letter but in my case it says it may be an acceptance letter which is not true.
5. After taking photo one time you can,t retake the photo before submit. For that you have to refresh the page.
6. Sometime when I upload image and proceed it says like " you can,t upload same image " although image is not same or it says like " binary data is missing" but the image was successfully taken and clearly visible.
7. Live chat support for education discount or email support is missing. (don't know where they solve education related issue in general support email but in education discount page there is no support email . Only you have a ticket creation link and there you found a AI chat bot which does nothing)
8. Although I had done Email verify. In my id there is my name and roll is visible .... still too much trouble for me. Also my id contains a qr code if you scan this you can verify if i an a student or not.

I think, we don't deserve this type of drawback from a reputed company like GitHub. Especially for student who will be the future GitHub user.

And ya I completely understand that some will come and provide fake id but this is not a solution to stop them. For that They may select a collage student from particular collage to verify their collage students who applied for student benefit. Also a upper level verification need to be done by GitHub Student Discount officials.

If GitHub Officials give us permission we may work on this 🙂 (like open source or a portal where we apply with our best projects and GitHub chooses us to work) . Also this we solve the problem and we have a good work history / contribution for our CV. 🙂

Thanks,
Subhajit 🙂

**Comment by samus-aran** [comment, marked as answer]
Hi @MSubhajitIND 👋 

We appreciate your feedback and this has been shared with the internal team 👍 Let me address some of your points here:

> 1. Upload document option mission ... we have to take photo and it's hard to get a proper image of id.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.

> 2. Sometimes we have to upload both side of our id but we can upload only 1 image

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

> 3. In case of mobile phone user when you tried to flip camera it would not work. Front camera is very bad to upload document images also hard to take image of id properly. ( this issue is not solved more than 1 year ... i saw past tickets in portal )

We are aware of this issue. Are you able to share your phone's model + OS?

> 4. Although there is no keyword in my ID which seems to be an acceptance letter but in my case it says it may be an acceptance letter which is not true.

This is in error. The internal team are aware of this. 🙂 

> 5. After taking photo one time you can,t retake the photo before submit. For that you have to refresh the page.

Thank you for this feedback.

> 6. Sometime when I upload image and proceed it says like " you can,t upload same image " although image is not same or it says like " binary data is missing" but the image was successfully taken and clearly visible.

Sometimes image detection isn't 100% but this issue may arrive if the user hadn't refreshed after setting up 2FA. Regarding to the binary data is missing, this may occur but we'll share this instance with the team also.

> 7. Live chat support for education discount or email support is missing. (don't know where they solve education related issue in general support email but in education discount page there is no support email . Only you have a ticket creation link and there you found a AI chat bot which does nothing)

We have made changes over the past year and improved documentation and the application process itself. A lot of our support enquiries can be self-served and we are working to improve this further. We direct users to utilise the community more and share knowledge on all things GitHub Education but sometimes users require explicit help. That is why our Community Managers are available to assist and direct queries more efficiently.

> 8. Although I had done Email verify. In my id there is my name and roll is visible .... still too much trouble for me. Also my id contains a qr code if you scan this you can verify if i an a student or not. 

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

**Reply by MSubhajitIND** [reply]
> Hi @MSubhajitIND 👋 
> 
> We appreciate your feedback and this has been shared with the internal team 👍 Let me address some of your points here:
> 
> > 1. Upload document option mission ... we have to take photo and it's hard to get a proper image of id.
> 
> It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.
> 
> > 2. Sometimes we have to upload both side of our id but we can upload only 1 image
> 
> - [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)
> 
> > 3. In case of mobile phone user when you tried to flip camera it would not work. Front camera is very bad to upload document images also hard to take image of id properly. ( this issue is not solved more than 1 year ... i saw past tickets in portal )
> 
> We are aware of this issue. Are you able to share your phone's model + OS?
> 
> > 4. Although there is no keyword in my ID which seems to be an acceptance letter but in my case it says it may be an acceptance letter which is not true.
> 
> This is in error. The internal team are aware of this. 🙂 
> 
> > 5. After taking photo one time you can,t retake the photo before submit. For that you have to refresh the page.
> 
> Thank you for this feedback.
> 
> > 6. Sometime when I upload image and proceed it says like " you can,t upload same image " although image is not same or it says like " binary data is missing" but the image was successfully taken and clearly visible.
> 
> Sometimes image detection isn't 100% but this issue may arrive if the user hadn't refreshed after setting up 2FA. Regarding to the binary data is missing, this may occur but we'll share this instance with the team also.
> 
> > 7. Live chat support for education discount or email support is missing. (don't know where they solve education related issue in general support email but in education discount page there is no support email . Only you have a ticket creation link and there you found a AI chat bot which does nothing)
> 
> We have made changes over the past year and improved documentation and the application process itself. A lot of our support enquiries can be self-served and we are working to improve this further. We direct users to utilise the community more and share knowledge on all things GitHub Education but sometimes users require explicit help. That is why our Community Managers are available to assist and direct queries more efficiently.
> 
> > 8. Although I had done Email verify. In my id there is my name and roll is visible .... still too much trouble for me. Also my id contains a qr code if you scan this you can verify if i an a student or not. 
> 
> As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. 

Thank you for your thorough response and for considering my feedback.

My mobile OS is MIUI and brand is POCO by Xiaomi.

**Comment by rachwhatsit** [comment]
I've been struggling with this process off and on for weeks. Thanks for so clearly detailing the problem and to the Github Education for a thoughtful response. 

I am struggling with the same issues with my phone. I am only able to use the front camera. My mobile OS is a iphone 11.

**Reply by MSubhajitIND** [reply]
If you are still in struggle do what I did, I mentioned above. 

Just download camo studio in you ios  and you pc and in laptop while uploading the id choose the Camo Studio camera.

https://apps.apple.com/us/app/camo-studio-stream-record/id6450313385

( if you have mac and ios both , i mean in apple ecosystem you can use your iphone camera as a laptop camera without additional software as of my knowledge)

If you are still in problem please let me know I will share a video in this post for future references who face the same problem.
